contrib/masterfiles-stage/README.org

```diff
@@ -172,11 +211,26 @@ bundle agent stage_masterfiles
 }
 #+end_src
 
+If the params file is placed in the default location
+`/opt/cfengine/dc-scripts/params.sh`, and if the value of `MASTERDIR`
+is specified in the params file, the CFEngine code can be as simple as:
+
+#+begin_src cfengine3
+bundle agent stage_masterfiles
+{
+  commands:
+    policy_server::
+      "/var/cfengine/bin/masterfiles-stage.sh";
+}
+#+end_src
+
+
 * Phased Rollout with Multiple Staged Masterfiles
 
 The `masterfiles-stage.sh` script can support an implementation for
 phased rollout. Phased Rollout is the process of releasing a *policy*
 change to a population in a controlled manner.
+(GIT_POLICY_CHANNELS is one such implementation.)
```

Comment:
I do think it would be nice to include more docs on your GIT_POLICY_CHANNELS and that doc was a good start.